.scala-steward.conf

```
@@ -1 +1,33 @@
-updatePullRequests = false
\ No newline at end of file
+<<<<<<< HEAD
+updatePullRequests = false
+=======
+allowedUpdates: [
+  // Apache Derby above 10.14 requires Java 9+. See
+  // https://db.apache.org/derby/derby_downloads.html
+  { groupId: "org.apache.derby", artifactId: "derby", version: "10.14." }
+]
+updates.ignore = [
+  // explicit updates
+  { groupId = "com.typesafe.akka" }, // Akka & Akka HTTP
+  { groupId = "org.scalameta", artifactId = "scalafmt-core" },
+  // these will get updated along with jackson-databind, so no need to update them
+  // separately
+  { groupId = "com.fasterxml.jackson.module", artifactId = "jackson-module-parameter-names" },
+  { groupId = "com.fasterxml.jackson.module", artifactId = "jackson-module-paranamer" },
+  { groupId = "com.fasterxml.jackson.module", artifactId = "jackson-module-scala" },
+  { groupId = "com.fasterxml.jackson.core", artifactId = "jackson-annotations" },
+  { groupId = "com.fasterxml.jackson.core", artifactId = "jackson-core" },
+  { groupId = "com.fasterxml.jackson.dataformat", artifactId = "jackson-dataformat-cbor" },
+  { groupId = "com.fasterxml.jackson.datatype", artifactId = "jackson-datatype-jsr310" },
+  { groupId = "com.fasterxml.jackson.datatype", artifactId = "jackson-datatype-jdk8" },
+  { groupId = "com.fasterxml.jackson.datatype", artifactId = "jackson-datatype-pcollections" },
+  { groupId = "com.fasterxml.jackson.datatype", artifactId = "jackson-datatype-guava" }
+]
+
+updates.pin = [
+  # To be updated in tandem with upstream Akka
+  { groupId = "com.fasterxml.jackson.core", artifactId = "jackson-databind", version = "2.11." }
+]
+
+updatePullRequests = false
+>>>>>>> 1a87308db9... Bump Jackson to 2.11.x
```

```suggestion
updatePullRequests = false
```

Is scala-steward checking non-`master` branches? IDK... in any case, I wouldn't care about it on this PR.